crates/gitbutler-tauri/Cargo.toml

```diff
@@ -29,6 +29,7 @@ backoff = "0.4.0"
 backtrace = { version = "0.3.71", optional = true }
 chrono = { version = "0.4.37", features = ["serde"] }
 console-subscriber = "0.2.0"
+crossbeam-channel = "0.5.12"
```

Comment:
Is there a reason for crossbeam channels vs using tokio's channels?

Reply:
Actually, `tokio` channels might be usable here and I didn't try. It would certainly be preferable if it worked.

Reply:
Let's try tokio then, lmk if it doesn't work for some reason. Just trying to reduce our number of dependencies.

Reply:
That's probably fine, we can move more toward async over time.